tests/queries/test_query_mimicry_mashup.py

```diff
@@ -0,0 +1,58 @@
+""" Unit tests for MimicryMacroMarketMashup queries.
+
+Copyright (c) 2021-, Tellor Development Community
+Distributed under the terms of the MIT License.
+"""
+from eth_abi import decode_abi
+
+from telliot_feeds.queries.mimicry.mimicry_macro_market_mash_up import MimicryMacroMarketMashup
+
+# example data from spec example
+# see: https://github.com/tellor-io/dataSpecs/blob/main/types/MimicryMacroMarketMashup.md
+metric = "market-cap"
```

Comment:
Can import these vars from `src/telliot_feeds/feeds/mimicry/mimicry_macro_market_mashup_feed_example.py` and rename originals to all uppercase as constants?